### ISSUE TITLE
Explicit setting of permissions required

### ISSUE THREAD

**Issue by amorriscode** [PlasmoHQ/storage#4]
Hey there! 👋🏻 

I tried using the storage library for a Chrome extension but noticed that the data wasn't synced between my content script and my popup. I was able to resolve the problem by modifying my permissions in the `package.json`:

```json
  "manifest": {
    "host_permissions": [
      "https://*/*"
    ],
    "permissions": [
      "storage"
    ]
  }
```

The documentation for the library implies that this should be handled automatically but it didn't work for me. 🤔

**Comment by ColdSauce** [comment]
Hey @amorriscode thanks for flagging this issue. I just fixed it in https://github.com/PlasmoHQ/plasmo/commit/cee085075ea888a439b6b28ae16d8c96d9f820ea

Will publish soon!

**Comment by ColdSauce** [comment]
Should be fixed in `0.42.7` so I'll close this issue. Let me know if you're still having problems!
